lib/services/identity.go

```diff
@@ -199,6 +201,30 @@ type Identity interface {
 	// DeleteSignupToken deletes signup token from the storage
 	DeleteSignupToken(token string) error
 
+	// upserts a U2F challenge for a new user corresponding to the token
+	UpsertU2fRegisterChallenge(token string, u2fChallenge *u2f.Challenge) (error)
+
+	// returns a U2F challenge for a new user corresponding to the token
```

Comment:
go comments begin with the function names according to this guideline that we follow

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences